main.js

```diff
@@ -0,0 +1,120 @@
+/*Heather Sleyster
+sleyster_a02b
+INFO 2134
+Thoendel
+03/17/2020
+*/
+window.addEventListener('load', function() {
+/*These statements assign the Form element to the form variable 
+and the select element named 'meal' to the select variable.
+*/
+const form = document.getElementById('contactForm');
+const meal = document.getElementsByName('meal');
+
+/*This if statement checks the value of the selected meal and if
+it is empty removes the 'visible' class from the form. If breakfast
+is selected the visible class will be added to that option only, if
+lunch is selected the visible class will be added to that option only,
+if dinner is selected the visible class will be added to that option.
+*/
+
+if (meal.value == '') {
+    form.classList.remove('visible');
+}
+    else if (meal.value == 'breakfast'){
+        const breakfastOptions = getElementById('breakfastOptions');
+        breakfastOptions.classList.add('visible');
+    }
+        else if (meal.value == 'lunch'){
+            const lunchOptions = getElementById('lunchOptions');
+            lunchOptions.classList.add('visible');
+        }
+            else if (meal.value == 'dinner') {
+                const dinnerOptions = getElementById('dinnerOptions');
+                dinnerOptions.classList.add('visible');
+            }
+/*These statements assign the elements with id values of firstNameHolder and 
+lastNameHolder to the variables firstName and lastName, the input id to the 
+variable input, the element with id ageHolder to the age variable, the element 
+with the id emailAddress to the emailAddress variable, the element with name of
+mealOptions to the mealOption variable, the element with id of mealRating to the
+variable mealRating.
+*/
+const firstName = document.getElementById('firstNameHolder');
+const lastName = document.getElementById('lastNameHolder');
+const input = document.getElementById('input');
+const age = document.getElementById('ageHolder');
+const emailAddress = document.getElementById('emailAdress');
+const mealOption = document.getElementsByName('mealOptions');
+const mealRating = document.getElementById('mealRating');
+/*This is the event listener to listen for the blur event and ensures that the 
+text in the respective field's  of firstName and lastName first character stays 
+upper case and all of the other characters stay lower case it also checks the input of
+the age element to see if it is a number and if it is not it will clear the input value
+and set the value to empty then focus the cursor back to the ageHolder element.
+*/
+input.addEventListener('blur', function() {
+firstName.input.toLowerCase();
+firstName.input[0].toUpperCase();
+lastName.input.toLowerCase();
+lastName.input[0].toUpperCase();
+if(typeof(age.input) != "number"){
+    alert('Please input only a number for age');
+    age.setAttribute('input', '');
+    age.focus();
```

Comment:
again - I'd have a separate listener here specifically for age. != "number" means you are checking to see if the value is literally the string "number" - check out the built-in function isNan() instead...